new/MDDL001.xml

```diff
@@ -0,0 +1,273 @@
+                                            <num value="20">፳</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="155r" to="162v"></locus>
+                                        </item>
+                                        <item xml:id="q22" n="21">
+                                            <num value="1">፩</num>
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="163r" to="168v"></locus>
+                                        </item>
+                                         </list>
+                                </collation>
+                                
+                                <condition key="good">The cover of the spine is missing.</condition>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="30">
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                                             <ab type="ruling">The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <locus from="5ra" to="165ra"></locus>
+                                <desc> Good hand.
+                                </desc>
+                                <seg type="ink">Black, red.</seg>
+                                <date notBefore="1800" notAfter="1899"/>
+                            </handNote>
+                            
+                                 <handNote script="Ethiopic" xml:id="h2">
+                                <locus from="165ra" to="166vb"></locus>
+                                <desc> Irregular hand.
+                                </desc>
+                                <seg type="ink">Black, red.</seg>
+                                <date notBefore="1800" notAfter="1899"/>
+                            </handNote>
+                            
+                        </handDesc>
+                        <additions>
+                            <list>
+                               
+                                <item xml:id="e1">
+                                    <locus target="#162vb"></locus>
+                                    <desc>The name of the commissioner was Walda ʾAmlāk, but another name (Gabra Māryām) has been written above it.
```

Comment:
It is OK for now, but eventually person records for the commissioner and owners should be created and linked here through `<persName ref="PRS1111111" role="owner"/>` tag